source/guides/network/ipxe-install.rst

```diff
@@ -49,8 +49,10 @@ shown in figure 2.
 
    Figure 2: Network topology.
 
-Your PXE client must have a boot order where the network boot option is
-prioritized before the disk boot option.
+Your PXE client system must meet the requirements to run |CL| and have a boot
+order where the network boot option is prioritized before the disk boot
+option. To determine if your PXE client system meets the minimum requirements
+for |CL|, review the :ref:`compatibility-check`.
```

Comment:
The new paragraph gives a lot more context to the network environment--thanks!  I think this paragraph would be work better if it appeared immediately after **Prerequsites**. Reason:  If I want to set up a PXE boot environment, I'd want to know if my PXE-client meets the minimum requirements *before* I connect the PXE server and PXE clients.

Reply:
I definitely agree with you but as I read the document, it seemed more focused on the server side of the equation so I left it in it's original location since many clients can use the server.  I'm easy though so I can move it up if you want me to.

Reply:
You *did* move the new paragraph up. I like it. Let's ship it!